fairseq/models/wav2vec/wav2vec2.py

```diff
@@ -1165,7 +1165,7 @@ def upgrade_state_dict_named(self, state_dict, name):
 
 
 class ConformerEncoder(TransformerEncoder):
-    def build_encoder_layer(self, args):
+    def build_encoder_layer(self, args, *largs, **kvargs):
```

Comment:
Do we need `*largs`  ?

 It might be simpler to not change `ConformerEncoder` at all but to change the base class - `TransformerEncoder.build_encoder_layer(self, args, **kwargs)`.  And use, `layer_idx` from `kwargs` in "trf_adp".